.swiftlint.yml

```diff
@@ -18,16 +18,17 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
 force_cast: warning # implicitly. Give warning only for force casting
 
 force_try:
-severity: warning # explicitly. Give warning only for force try
+  severity: warning # explicitly. Give warning only for force try
```

Comment:
Should this be on the same line, or is it like an if block?

Reply:
I think this can be on the same line. I think this was taken from their demo code and they were showing the different ways the settings can be changed. I'll put it on the same line

Reply:
Was just curious!